esp/src/eclwatch/nls/es/hpcc.js

```diff
@@ -219,6 +236,7 @@ define(
     FileScopes: "Ámbitos de Archivos",
     FileSize: "Tamaño de archivo",
     FilesNoPackage: "Archivos sin ",
+    FilesWarning: "La cantidad de archivos devueltos es demasiado grande. Solo se devolvieron los primeros 100.000 archivos ordenados por fecha / hora modificada. Si desea limitar los resultados, configure un filtro.",
```

Comment:
What's the original text for  FilesWarning in the english. why  is 100.000 using floating point?

Reply:
The number of files returned is too large. Only the first 100,000 files sorted by date/time modified were returned. If you wish to limit results, set a filter.